core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.template.html

```diff
@@ -28,7 +28,8 @@ <h3>Upload Thumbnail</h3>
     </div>
     <div ng-show="uploadedImage">
       <div ng-if="allowedBgColors">
-        <div>Choose background color:</div>
+        <div ng-if="allowedBgColors.length > 1">Choose background color:</div>
+        <div ng-if="allowedBgColors.length === 1">This thumbnail will have a default background color:</div>
```

Comment:
I'm also thinking, why do we need to tell them what the background colour is if they can already see it.

Maybe: Please check that your thumbnail has the following background color: [inline image]. If not, please ensure that you are uploading an image with transparent background. Thanks!

(And highlight this in yellow, similar to what we do on the regular image modal, so that it stands out.)

Reply:
Did you mean something like this?
![updated_message](https://user-images.githubusercontent.com/11008603/80908316-8361cb80-8d3c-11ea-94da-662a5716d1be.png)

Reply:
Yes, looks good, thanks!

Reply:
Done.